docs/src/index.rst

```diff
@@ -0,0 +1,23 @@
+*****************************************************************
+A Getting Started Guide to the Genomic Epidemiology of SARS-CoV-2
+*****************************************************************
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   :caption: Table of contents
+
+   self
+   analysis/index
+   visualization/index
+   advanced/index
+
+This template and tutorial will walk you through the process of running a basic phylogenetic analysis on SARS-CoV-2 data.
```

Comment:
In the GitHub Pages version of the tutorial, this content was written to precede the table of contents, as a way of introducing what was to come. We should move the table of contents below this intro, so the content will continue to make sense.

Reply:
I think the GitHub pages version is superior here, and the more we can make the RTD flow like that the better. For the TOC, it's really nice how the GitHub pages version (a) includes a sentence explaining each section and (b) lists all the pages in each section.